### PR TITLE
Added FOXEER_F745_AIO target

### PR DESCRIPTION
**Pull request by Johnsnow-foxeer**
Added FOXEER_F745_AIO target

### CODE REVIEW

**Comment by Johnsnow-foxeer** [comment]
Please make me as the code owner, thank you!

**Comment by Johnsnow-foxeer** [comment]
Please help if you are free, thank you, @mikeller  @McGiverGim  @haslinghuis

File: configs/default/FOXE-FOXEERF745AIO
Comment (Asizon) [inline configs/default/FOXE-FOXEERF745AIO:124]: Is this all in one aio board? Else esc protocol should be NONE

File: configs/default/FOXE-FOXEERF745AIO
Comment (Asizon) [inline configs/default/FOXE-FOXEERF745AIO:123]: Why you are enabling dhost bidirectional as default?

**Review by Asizon — CHANGES_REQUESTED** [review]

File: configs/default/FOXE-FOXEERF745AIO
Comment (Johnsnow-foxeer) [reply configs/default/FOXE-FOXEERF745AIO]: It is a FC&ESC AIO board, I just changed to NONE

File: configs/default/FOXE-FOXEERF745AIO
Comment (Johnsnow-foxeer) [reply configs/default/FOXE-FOXEERF745AIO]: Sorry, my fault, changed to OFF

File: configs/default/FOXE-FOXEERF745AIO
Comment (Asizon) [reply configs/default/FOXE-FOXEERF745AIO]: Thankss, is most safely. But for fc with integrated esc we allow target maintaienrs to set dshot protocol.

File: configs/default/FOXE-FOXEERF745AIO
Comment (Asizon) [inline configs/default/FOXE-FOXEERF745AIO:108]: This airmode disabling should be removed

**Review by Asizon — CHANGES_REQUESTED** [review]

File: configs/default/FOXE-FOXEERF745AIO
Comment (Asizon) [inline configs/default/FOXE-FOXEERF745AIO:123]: Remove entire line please, it is off by default.

**Review by Asizon — CHANGES_REQUESTED** [review]

File: configs/default/FOXE-FOXEERF745AIO
Comment (Asizon) [inline configs/default/FOXE-FOXEERF745AIO:124]: This is also none by default

**Review by Asizon — CHANGES_REQUESTED** [review]

File: configs/default/FOXE-FOXEERF745AIO
Comment (Johnsnow-foxeer) [reply configs/default/FOXE-FOXEERF745AIO]: Changed

File: configs/default/FOXE-FOXEERF745AIO
Comment (Johnsnow-foxeer) [reply configs/default/FOXE-FOXEERF745AIO]: Removed

**Comment by Asizon** [comment]
Perfect thanks, then are you the current Foxeer maintainer? We need to update anithing?

**Comment by Johnsnow-foxeer** [comment]
Thank you so much, if I am already the code owner, ignore it, thank you.

**Comment by SteveCEvans** [comment]
@Johnsnow-foxeer In order to get support for this hardware, please provide samples of the FC to @mikeller, @haslinghuis and myself. You can contact us for postal addresses on slack.

**Comment by SteveCEvans** [comment]
I should add that it's hard work testing changes against all hardware, and if we can't test your hardware then the probability of something breaking is clearly increased. Your hardware is at the bleeding edge and therefore not so tried and tested, so extra diligence is called for.

**Comment by DusKing1** [comment]
Also @pdw and I need some samples to test please DM us through previous contact.

**Comment by DusKing1** [comment]
Also, before merging a squashing and/or force push is needed.

**Comment by Johnsnow-foxeer** [comment]
> @Johnsnow-foxeer In order to get support for this hardware, please provide samples of the FC to @mikeller, @haslinghuis and myself. You can contact us for postal addresses on slack.

Got it, I prepare the samples, thank you

**Comment by DusKing1** [comment]
> Please make me as the code owner, thank you!

Can you elaborate yourself? GitHub is not a place where business in Shenzhen where back-stabbing and betrayal is a common thing, we take the code and related matters seriously here. I have received same requests that asking me to be someone's code owner and without any further update or elaboration the same person ask not let me be their code owner, like https://github.com/betaflight/unified-targets/pull/500#issuecomment-903209544 and later https://github.com/betaflight/unified-targets/pull/501#issue-976783518

I don't wanna over-exaggerate the impact from those half-drama-ish comments/matters/request, but to be honest those bullshits have seriously affected my personal life and my mood. I have to clarify one thing, that we here contributors are volunteering our own spare time, no matter paid by vendors or not, to check your vendors' PRs, feedback, even those non-sense drama issue that you guys post here. So we need you guys from vendors, especially from China, to take those things seriously. If not, please get away from betaflight.

File: configs/default/FOXE-FOXEERF745AIO
Comment (mikeller) [inline configs/default/FOXE-FOXEERF745AIO:3]: This will not work as a target definition as the file name is incorrect. Please read and follow the instructions in https://github.com/betaflight/betaflight/blob/master/docs/TargetMaintenance/CreatingAUnifiedTarget.md hire somebody who has the skills to do so.

**Review by mikeller — CHANGES_REQUESTED** [review]

File: configs/default/FOXE-FOXEERF745AIO
Comment (Johnsnow-foxeer) [reply configs/default/FOXE-FOXEERF745AIO]: Fixed the file name, sorry for issue, I missed ".config"

**Comment by Asizon** [comment]
@Johnsnow-foxeer  Thanks for the changes. Looks like we need to do a codeowner change again, on August 7th it was changed to Dusking.
On the other hand, I would also like to receive a Foxeer enginering sample, it will be apreciated.

**Comment by Johnsnow-foxeer** [comment]
> @Johnsnow-foxeer Thanks for the changes. Looks like we need to do a codeowner change again, on August 7th it was changed to Dusking. On the other hand, I would also like to receive a Foxeer enginering sample, it will be apreciated.

I am happy to provide the samples... how can I know your address? I have other developers's address already.

**Comment by DusKing1** [comment]
No, @Johnsnow-foxeer you didn't promise to ship samples to @pdw and me

**Review by haslinghuis — APPROVED** [review]

**Review by limonspb — APPROVED** [review]

**Review by blckmn — APPROVED** [review]

**Pull request merged**
